packages/cactus-core-api/src/main/json/openapi.json

```diff
@@ -1,751 +1,793 @@
+            "minLength": 1,
+            "maxLength": 1024,
+            "nullable": false
+          }
+        }
+      },
+      "GetKeychainEntryResponse": {
+        "type": "object",
+        "required": ["key", "value"],
+        "properties": {
+          "key": {
+            "type": "string",
+            "description": "The key that was used to retrieve the value from the keychain.",
+            "minLength": 1,
+            "maxLength": 1024,
+            "nullable": false
+          },
+          "value": {
+            "type": "string",
+            "description": "The value associated with the requested key on the keychain.",
+            "minLength": 0,
+            "maxLength": 10485760,
+            "nullable": false
+          }
+        }
+      },
+      "SetKeychainEntryRequest": {
+        "type": "object",
+        "required": ["key", "value"],
+        "properties": {
+          "key": {
+            "type": "string",
+            "description": "The key for the entry to set on the keychain.",
+            "minLength": 1,
+            "maxLength": 1024,
+            "nullable": false
+          },
+          "value": {
+            "type": "string",
+            "description": "The value that will be associated with the key on the keychain.",
+            "minLength": 0,
+            "maxLength": 10485760,
+            "nullable": false
+          }
+        }
+      },
+      "SetKeychainEntryResponse": {
+        "type": "object",
+        "required": ["key"],
+        "properties": {
+          "key": {
+            "type": "string",
+            "description": "The key that was used to set the value on the keychain.",
+            "minLength": 1,
+            "maxLength": 1024,
+            "nullable": false
+          }
+        }
+      },
+      "DeleteKeychainEntryRequestV1": {
+        "type": "object",
+        "required": ["key"],
+        "properties": {
+          "key": {
+            "type": "string",
+            "description": "The key for the entry to check the presence of on the keychain.",
+            "minLength": 1,
+            "maxLength": 1024,
+            "nullable": false
+          }
+        }
+      },
+      "DeleteKeychainEntryResponseV1": {
+        "type": "object",
+        "required": ["key"],
+        "properties": {
+          "key": {
+            "type": "string",
+            "description": "The key that was deleted from the keychain.",
+            "minLength": 1,
+            "maxLength": 1024,
+            "nullable": false
+          }
+        }
+      },
+      "HasKeychainEntryRequestV1": {
+        "type": "object",
+        "required": ["key"],
+        "properties": {
+          "key": {
+            "type": "string",
+            "description": "The key to check for presence in the keychain.",
+            "minLength": 1,
+            "maxLength": 1024,
+            "nullable": false
+          }
+        }
+      },
+      "HasKeychainEntryResponseV1": {
```

Comment:
@Leeyoungone Please model this one after the similarly named endpoint response object for the object store plugin (you can look at the definition of it in this same file)

```
"HasObjectResponseV1": {
        "type": "object",
        "required": ["key", "isPresent", "checkedAt"],
        "properties": {
          "key": {
            "type": "string",
            "description": "The key that was used to check the presence of the value in the object store.",
            "minLength": 1,
            "maxLength": 1024,
            "nullable": false
          },
          "checkedAt": {
            "type": "string",
            "description": "Date and time encoded as JSON when the presence check was performed by the plugin backend.",
            "nullable": false
          },
          "isPresent": {
            "type": "boolean",
            "description": "The boolean true or false indicating the presence or absence of an object under 'key'.",
            "nullable": false
          }
        }
      },
```